raiden/transfer/utils.py

```diff
@@ -107,11 +113,13 @@ def get_event_with_balance_proof_by_locksroot(
     balance proof.
     """
     return storage.get_latest_event_by_data_field({
-        'balance_proof.chain_id': canonical_identifier.chain_identifier,
-        'balance_proof.token_network_identifier': to_checksum_address(
+        'balance_proof.canonical_identifier.chain_identifier':
+        str(canonical_identifier.chain_identifier),
```

Comment:
This was a bit confusing, can you change this to: 
```suggestion
        'balance_proof.canonical_identifier.chain_identifier': str(
            canonical_identifier.chain_identifier
        ),
```

Reply:
resolved